improver/utilities/mathematical_operations.py

```diff
@@ -638,14 +651,306 @@ def process(
+                        "match the time coordinate of the mean cube. I.e. The same mean"
+                        "cube that was used to calculate the anomaly."
+                        "The following values were found: "
+                        f"anomaly cube reference epoch: {anomaly_cube.coord('reference_epoch').points},"
+                        f"mean cube time: {mean_cube.coord('time').points},"
+                        f"anomaly cube reference epoch bounds: {anomaly_cube.coord('reference_epoch').bounds},"
+                        f"mean cube time bounds: {mean_cube.coord('time').bounds}"
+                    )
+
+        if errors:
+            raise ValueError("\n".join(errors))
+
+    @staticmethod
+    def calculate_forecast_value(
+        anomaly_cube: Cube, mean_cube: Cube, std_cube: Optional[Cube] = None
+    ) -> ndarray:
+        """Calculate forecast value(s) from the input cubes."""
+        if std_cube:
+            forecast_value = (anomaly_cube.data * std_cube.data) + mean_cube.data
+        else:
+            forecast_value = anomaly_cube.data + mean_cube.data
+
+        return forecast_value
+
+    @staticmethod
+    def _update_cube_name_and_units(
+        output_cube: Cube, mean_cube: Cube, standardized_anomaly: bool = False
+    ) -> None:
+        """This method removes the portion of the name of the output cube
+        cube that indicates that it is an anomaly and converts units to be dimensional
+        (i.e. not '1') if a standardized anomaly is used. The cube is modified in place.
+
+        Args:
+            output_cube:
+                The cube to be updated.
+            standardized_anomaly:
+                Flag indicating if the forecast values are calculated using
+                standardized anomalies. If True, a "_standardized_anomaly" suffix is
+                removed from the name. If False, an "_anomaly" suffix is removed from
+                the name.
+        """
+        suffix = "_standardized_anomaly" if standardized_anomaly else "_anomaly"
+
+        try:
+            output_cube.standard_name = output_cube.standard_name.replace(suffix, "")
+        except ValueError:
+            output_cube.long_name = output_cube.long_name.replace(suffix, "")
+
+        output_cube.units = mean_cube.units
+
+    @staticmethod
+    def _remove_reference_epoch_metadata(output_cube: Cube) -> None:
+        """Remove the reference epoch coordinate and cell method from the output cube
+        if present. Such metadata would be present on any anomaly cube processed through
+        the plugin CalculateClimateAnomalies. This is done to ensure that the output
+        cube does not contain any unnecessary metadata related to the anomaly
+        calculation. The metadata is removed in place.
+
+        Args:
+            output_cube:
+                The cube from which to remove the reference epoch metadata.
+        """
+        output_cube.remove_coord("reference_epoch")
```

Comment:
```suggestion
        if output_cube.coords("reference_epoch"):
            output_cube.remove_coord("reference_epoch")

        output_cube.cell_methods = tuple(
            cm for cm in output_cube.cell_methods if cm.method != "anomaly"
        )
```
I think that the `reference_epoch` coordinate should always be present, but this provides some security in case it isn't. A test will need adding for this line.

Reply:
Added with a test. Thank you.